yt_dlp/extractor/livestream.py

```diff
@@ -22,12 +22,12 @@
 
 class LivestreamIE(InfoExtractor):
     IE_NAME = 'livestream'
-    _VALID_URL = r'https?://(?:new\.)?livestream\.com/(?:accounts/(?P<account_id>\d+)|(?P<account_name>[^/]+))/(?:events/(?P<event_id>\d+)|(?P<event_name>[^/]+))(?:/videos/(?P<id>\d+))?'
+    _VALID_URL = r'https?://(?:new\.)?livestream.com/(?:(?:accounts/(?P<account_id>\d+))|(?P<account_name>[^/]+))(?:(?:/events/(?P<event_id>\d+))|(?:/(?P<event_name>[^/]+)))?(?:/videos/(?P<id>\d+))?'
```

Comment:
Hard to read. could u split it up into multiple lines?

Reply:
done